dist/constants.json

```diff
@@ -8689,225 +8689,225 @@
+        "label": "Palo Alto VA Medical Center-Livermore"
       },
       {
         "value": "640BY",
-        "label": "SAN JOSE VA CLINIC"
+        "label": "San Jose VA Clinic"
       },
       {
         "value": "570GA",
-        "label": "MERCED VA CLINIC"
+        "label": "Merced VA Clinic"
       },
       {
         "value": "570GB",
-        "label": "TULARE VA CLINIC"
+        "label": "Tulare VA Clinic"
       },
       {
         "value": "612A4",
-        "label": "SACRAMENTO VA MEDICAL CENTER"
+        "label": "Sacramento VA Medical Center"
       },
       {
         "value": "612B4",
-        "label": "REDDING VA CLINIC"
+        "label": "Redding VA Clinic"
       },
       {
         "value": "612BY",
-        "label": "OAKLAND VA CLINIC"
+        "label": "Oakland VA Clinic"
       },
       {
         "value": "612GD",
-        "label": "FAIRFIELD VA CLINIC"
+        "label": "Fairfield VA Clinic"
       },
       {
         "value": "612GE",
-        "label": "MARE ISLAND VA CLINIC"
+        "label": "Mare Island VA Clinic"
       },
       {
         "value": "612GF",
-        "label": "MARTINEZ VA MEDICAL CENTER"
+        "label": "Martinez VA Medical Center"
       },
       {
         "value": "612GG",
-        "label": "CHICO VA CLINIC"
+        "label": "Chico VA Clinic"
       },
       {
         "value": "612GH",
-        "label": "MCCLELLAN VA CLINIC"
+        "label": "McClellan VA Clinic"
       },
       {
         "value": "605GA",
-        "label": "VICTORVILLE CBOC"
+        "label": "Victorville VA Clinic"
       },
       {
         "value": "605GB",
-        "label": "MURRIETA CBOC"
+        "label": "Murrieta VA Clinic"
       },
       {
         "value": "605GC",
-        "label": "PALM DESERT CBOC"
+        "label": "Palm Desert VA Clinic"
       },
       {
         "value": "605GD",
-        "label": "CORONA CBOC"
+        "label": "Corona VA Clinic"
       },
       {
         "value": "605GE",
-        "label": "RANCHO CUCAMONGA CBOC"
+        "label": "Rancho Cucamonga VA Clinic"
       },
       {
         "value": "600GA",
-        "label": "ANAHEIM VETERANS HEALTH CLINIC"
+        "label": "Anaheim VA Clinic"
       },
       {
         "value": "600GB",
-        "label": "SANTA ANA VETERANS HEALTH CLINIC"
+        "label": "Santa Ana VA Clinic"
       },
       {
         "value": "600GC",
-        "label": "CABRILLO VETERANS HEALTH CLINIC"
+        "label": "Cabrillo VA Clinic"
       },
       {
         "value": "600GD",
-        "label": "SANTA FE SPRINGS VA CLINIC"
+        "label": "Santa Fe Springs VA Clinic"
       },
       {
         "value": "654GA",
         "label": "SIERRA FOOTHILLS VA CLINIC"
       },
       {
         "value": "662GE",
-        "label": "SAN BRUNO CBOC"
+        "label": "San Bruno VA Clinic"
       },
       {
         "value": "640HC",
-        "label": "MONTEREY VA CLINIC"
+        "label": "Major General William H. Gourley VA-DoD Outpatient Clinic"
```

Comment:
Let's confirm this name change.

Reply:
This page has a reference of **MONTEREY VA CLINIC** in the description txt (https://www.paloalto.va.gov/locations/monterey.asp):
> Parent Facility: The Monterey Clinic is part of the VA Palo Alto Health Care System.

So LGTM.